src/tile/tile.ts

```diff
@@ -44,11 +50,28 @@ export const getAllSymbols = (): Tile[] => {
   const neutralTile: NeutralTile = {
     symbol: "O",
     color: "grey",
+    playedBy: null,
+    lastPlayed: false,
   };
   return [...flattenedTiles, neutralTile];
 };
 
 export const renderTile = (tile: Tile): string => {
-  const dynamicChalk = chalk[tile.color];
-  return dynamicChalk(`${tile.symbol} `);
+  let dynamicChalk = chalk[tile.color];
+
+  if (tile.playedBy != null) {
+    if (tile.playedBy === "black") {
+      dynamicChalk = dynamicChalk.bgWhite.dim;
+    }
+
+    if (tile.playedBy === "white") {
+      dynamicChalk = dynamicChalk.bgWhite;
+    }
+  }
+
+  if (tile.lastPlayed === true) {
+    dynamicChalk = dynamicChalk.bold.underline;
+  }
+
+  return dynamicChalk(tile.symbol);
```

Comment:
```suggestion
  return dynamicChalk(`${tile.symbol} `);
```
Otherwise this is a regression